Source/ProgressionSystemRuntime/Public/Data/PSTypes.h

```diff
@@ -41,17 +41,9 @@ struct FPSRowData : public FTableRowBase
 	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category="C++")
 	float PointsToUnlock = 0.f;
 
-	/** Amount of points to gain after win */
-	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category="C++")
-	float WinReward = 0.f;
-
-	/** Amount of points to gain after draw */
-	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category="C++")
-	float DrawReward = 0.f;
-
-	/** Amount of points to gain after loss */
-	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category="C++")
-	float LossReward = 0.f;
+	/** The Progression End Game States */
+	UPROPERTY(EditDefaultsOnly, BlueprintReadWrite, Category="C++", meta = (DisplayName = "Progression End Game States"))
+	TMap<EEndGameState, float> ProgressionEndGameState;
```

Comment:
Current name and description sound too generic. It's unclear whether this property contains multipliers, scores, or something else without closely reading the code. Consider using a more descriptive name like `EndGameScoreValues`, and update the description to clarify that it holds base score values for each end-game result, which may be adjusted by difficulty level. This approach makes it clear that these are the initial scores (e.g., star ratings for a win).

### Expected result
```cpp
/** Base scores per end-game result before applying difficulty multiplier. E.g., the number of stars a player receives upon winning the game. 
 * If End-Game state is not matching with game result, 0 score will be granted by default. */
UPROPERTY(EditDefaultsOnly, BlueprintReadWrite, Category="C++", meta = (DisplayName = "End Game Score Values"))
TMap<EEndGameState, float> EndGameScoreValues;
```